meta.lib.spoofax/trans/libspoofax/analysis/constraint.str

```diff
@@ -119,3 +119,9 @@ rules
   get-project-constraint-analyses =
     prim("C_get_project_analyses")
 
+  /** Get analyzed ASTs for all resources in the project
+   *
+   * @type _ -> List(resource:String * asts:Term)
```

Comment:
`asts` should be `ast` (because _inside_ the list its singular).

Reply:
Done